### PR TITLE
MIMIR-114: Get owner for tbml tables

### PR DESCRIPTION
**Pull request by itemodahl**
Get sourcelist when registering a new tbml
Merge sourcelist with tbml data
Use sourcelist to show owner in update modal
Group sourcelist on owner id
Enable username and password fields
Send in user, password and owner id to backend

MIMIR-114

### CODE REVIEW

**Comment by itemodahl** [comment]
This code only support one owner. Support for multiple owners will come in the following PR's.

File: src/main/resources/lib/ssb/dataset/tbprocessor.ts [hunk 1]
```diff
@@ -29,32 +30,73 @@ export function getTbprocessor(content: Content<DataSource>, branch: string): Da
...
+      })
+    }
+    log.error(message)
   }
-  return data
+  return null
+}
+
+function getDataAndMetaData(content: Content<DataSource>): TbmlData | null {
+  const baseUrl: string = app.config && app.config['ssb.tbprocessor.baseUrl'] ? app.config['ssb.tbprocessor.baseUrl'] : 'https://i.ssb.no/tbprocessor'
```
Comment (ssb-cgn) [inline src/main/resources/lib/ssb/dataset/tbprocessor.ts:79]: Endre fallback to https://i.utv.ssb.no/tbprocessor or https://i.test.ssb.no/tbprocessor

File: src/main/resources/lib/ssb/statistic.ts [hunk 2]
```diff
@@ -68,17 +80,64 @@ export function setupHandlers(socket: Socket, socketEmitter: SocketEmitter): voi
...
+  return content ? getTbprocessor(content, 'master') : null
+}
+
 function prepStatistics(statistics: Array<Content<Statistics>>): Array<StatisticDashboard> {
   const statisticData: Array<StatisticDashboard> = []
   statistics.map((statistic: Content<Statistics>) => {
     const statregData: StatregData | undefined = statistic.data.statistic ? getStatregInfo(statistic.data.statistic) : undefined
-    if (statregData && statregData.nextRelease && moment(statregData.nextRelease).isSameOrAfter(new Date(), 'day')) {
+    const relatedTables: Array<TbmlSources> = sourceListFromStatistic(statistic)
+    //if (statregData && statregData.nextRelease && moment(statregData.nextRelease).isSameOrAfter(new Date(), 'day')) {
```
Comment (ssb-cgn) [inline src/main/resources/lib/ssb/statistic.ts:132]: Why is the check isSameOrAfter removed?

**Review by ssb-cgn — APPROVED** [review]
Approved with som comments :)

File: src/main/resources/lib/ssb/statistic.ts (hunk 2 above)
Comment (itemodahl) [reply src/main/resources/lib/ssb/statistic.ts]: I needed to get some content in the list. Should be changed back to normal the normal test.

File: src/main/resources/lib/ssb/dataset/tbprocessor.ts (hunk 1 above)
Comment (itemodahl) [reply src/main/resources/lib/ssb/dataset/tbprocessor.ts]: Default is set to test environment.

**Pull request merged**